HealthcareApp/src/HealthcareApi/Controllers/AuthController.cs

```diff
@@ -0,0 +1,39 @@
+using Application.Abstractions;
+using Application.DTOs.Login;
+using Application.DTOs.Register;
+using Microsoft.AspNetCore.Mvc;
+
+namespace HealthcareApi.Controllers;
+
+[Route("api")]
+public class AuthController : Controller
+{
+    private readonly IUserAuthenticationService _userAuthenticationService;
+
+    public AuthController(IUserAuthenticationService userAuthenticationService) =>
+        this._userAuthenticationService = userAuthenticationService;
+
+    [HttpPost("login")]
+    public async Task<IActionResult> Login([FromBody] LoginUserDTO loginUserDto)
+    {
+        if (!ModelState.IsValid)
+            return BadRequest(ModelState);
```

Comment:
Fluent validation middleware should validate the request automatically. This logic look redundant